src/openrct2/world/Footpath.cpp

```diff
@@ -514,7 +514,7 @@ static TileElement* footpath_connect_corners_get_neighbour(int32_t x, int32_t y,
  *
  *  rct2: 0x006A70EB
  */
-static void footpath_connect_corners(int32_t initialX, int32_t initialY, TileElement* initialTileElement)
+static void footpath_connect_corners(const CoordsXY& footpathPos, TileElement* initialTileElement)
```

Comment:
Can I be a pain and request this takes in a `PathElement*` instead of a `TileElement*`. It will reduce the AsPath() and mean the errors involved are higher up.

Reply:
Done